internal/manager/manager_tasks.go

```diff
@@ -418,3 +440,132 @@ func (s *Manager) StashBoxBatchPerformerTag(ctx context.Context, input StashBoxB
 
 	return s.JobManager.Add(ctx, "Batch stash-box performer tag...", j)
 }
+
+func (s *Manager) StashBoxBatchStudioTag(ctx context.Context, input StashBoxBatchTagInput) int {
+	j := job.MakeJobExec(func(ctx context.Context, progress *job.Progress) {
```

Comment:
Almost a straight up duplication of the above method. I would like to refactor them in a future update.